pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/TableViews.java

```diff
@@ -141,28 +146,44 @@ public TableView getExternalView(
   public String getSegmentsStatusDetails(
       @ApiParam(value = "Name of the table", required = true) @PathParam("tableName") String tableName,
       @ApiParam(value = "realtime|offline", required = false) @QueryParam("tableType") String tableTypeStr,
+      @ApiParam(value = "Show segments being replaced: true|false", required = false)
```

Comment:
(minor) For consistency
```suggestion
      @ApiParam(value = "Include replaced segments", required = false)
```